### PR TITLE
Caver-js transaction sample

### PR DESCRIPTION
**Pull request by terrikwak**
## Proposed changes

Removed "Sending KLAY at a glance" from Getting Started 
Added a simple tutorial for caver-js

Gitbook link:
https://app.gitbook.com/o/-LZ3M2TduHoL5IluFRpr/s/2zebjlN2CCetZOGScXQP/

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Comment by kjhman21** [comment]
@terrikwak You don't need to add .DS_Store files.

**Review by toniya-klaytn — COMMENTED** [review]
@terrikwak  In prompt.png ->  is it `node testcaver` or `node testcaver.js`

**Review by ayo-klaytn — COMMENTED** [review]
@terrikwak we can as well initialise the npm project like this

`npm init -y`

 to avoid answering questions and further explanation

File: docs/dapp/sdk/caver-js/sending-transaction.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+```
+async function testFunction() {
+// Read keystore json file
+  const fs = require('fs')
+	const keystore = fs.readFileSync('./keystore.json', 'utf8')
+	const password = await loadPassword()
+
+	// Decrypt keystore and create
+	const keyring = caver.wallet.keyring.decrypt(keystore, 'password')
+	console.log(keyring)
```
Comment (tantely-klaytn) [inline docs/dapp/sdk/caver-js/sending-transaction.md:140]: Should be changed to ```const keyring = caver.wallet.keyring.decrypt(keystore, password)```

**Review by tantely-klaytn — COMMENTED** [review]
Suggestions added in related sections

File: docs/dapp/sdk/caver-js/sending-transaction.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+### Prerequisites
+
+First install the following packages.
+* [Node.js](https://nodejs.org/en/download/) version ([14.16.0](https://nodejs.org/dist/latest-v14.x/))
+* [npm](https://www.npmjs.com/get-npm)
+* [nvm](https://github.com/nvm-sh/nvm)
+* [Solidity compiler](https://solidity.readthedocs.io/en/develop/installing-solidity.html)
+
+*Note:* If you get an `nmv: command not found` error after installing nvm, refer to this [troubleshooting guide](https://github.com/nvm-sh/nvm/issues/2060).
```
Comment (terrikwak) [inline docs/dapp/sdk/caver-js/sending-transaction.md:15]:
```suggestion
*Note:* If you get an `nvm: command not found` error after installing nvm, refer to this [troubleshooting guide](https://github.com/nvm-sh/nvm/issues/2060).
```

File: docs/dapp/sdk/caver-js/sending-transaction.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+```
+async function testFunction() {
+// Read keystore json file
+  const fs = require('fs')
+	const keystore = fs.readFileSync('./keystore.json', 'utf8')
+	const password = await loadPassword()
+
+	// Decrypt keystore and create
+	const keyring = caver.wallet.keyring.decrypt(keystore, 'password')
```
Comment (terrikwak) [inline docs/dapp/sdk/caver-js/sending-transaction.md:139]:
```suggestion
	const keyring = caver.wallet.keyring.decrypt(keystore, password)
```

File: docs/dapp/sdk/caver-js/sending-transaction.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+
+    // Add to caver.wallet
+	caver.wallet.add(keyring)
+
+	}
+```
+
+### 7. Send Transaction <a id="7.-send-transaction"></a> 
+
+We will now create a trasaction to transfer some KLAY. This type of transaction is called "value transfer transaction". Let's break down each parameter.
```
Comment (terrikwak) [inline docs/dapp/sdk/caver-js/sending-transaction.md:150]:
```suggestion
We will now create a transaction to transfer some KLAY. This type of transaction is called "value transfer transaction". Let's break down each parameter.
```

**Review by toniya-klaytn — APPROVED** [review]

**Comment by terrikwak** [comment]
@kjhman21 Could you take another look?

**Review by ayo-klaytn — APPROVED** [review]

**Review by toniya-klaytn — APPROVED** [review]

**Comment by kjhman21** [comment]
I can also see .DS_Store files in your commit.

**Pull request merged**